### ISSUE TITLE
Cannot do 'OR' queries

### ISSUE THREAD

**Issue by betseyb** [evangsherlock/nytimesarticle#1]
From the API dashboard, the api accepts queries like q="Obama OR Biden", but they return nothing when passed through your code.  It's not getting encoded correctly by requests, and when I muck around with trying to encode it myself, it just screws up the encoding of the rest of the request.

**Comment by handrew** [comment]
@betseyb , yeah it seems like the author merely string-formatted in the HTTP request.

**Comment by HLoho** [comment]
Lol if anyone sees this, to do OR queries, you should really put it in the fq field, because the q field can't handle Lucene syntax (OR, AND, &&, etc.).
